doc/releases/quincy.rst

```diff
@@ -5,6 +5,308 @@ Quincy
 Quincy is the 17th stable release of Ceph.  It is named after Squidward
 Quincy Tentacles from Spongebob Squarepants.
 
+v17.2.4 Quincy
+==============
+
+This is the fourth backport release in the Quincy series. We recommend
+that all users update to this release.
+
+Notable Changes
+---------------
+
+* Cephfs: The 'AT_NO_ATTR_SYNC' macro is deprecated, please use the standard
+  'AT_STATX_DONT_SYNC' macro. The 'AT_NO_ATTR_SYNC' macro will be removed in
+  the future.
+
+* OSD: The issue of high CPU utilization during recovery/backfill operations
+  has been fixed. For more details see: https://tracker.ceph.com/issues/56530.
+
+* Trimming of PGLog dups is now controlled by the size instead of the version.
+  This fixes the PGLog inflation issue that was happening when online
```

Comment:
Suggest s/that was happening//

Reply:
I personally think "that was happening" works better to clarify the sentence.